### PR TITLE
Feat/integrate dispute details

### PR DESCRIPTION
**Pull request by MichaelHorwitz**

### CODE REVIEW

File: frontend/src/lib/interfaces/dispute.ts [hunk 1]
```diff
@@ -23,7 +23,7 @@ export type DisputeResponse = {
   title: string;
   description: string;
   status: string;
-  date_created: string;
+  case_date: string;
```
Comment (ghost) [inline frontend/src/lib/interfaces/dispute.ts:26]: Did the contracts change?

File: frontend/src/app/disputes/[id]/page.tsx [hunk 2]
```diff
@@ -53,13 +55,15 @@ export default async function DisputePage({ params }: Props) {
   if (error || !data) {
     return <h1>{error}</h1>;
   }
-
+  var eviStr : string = data.evidence[0].toString()
+  eviStr = eviStr.split("/").pop() as string
```
Comment (ghost) [inline frontend/src/app/disputes/[id]/page.tsx:59]: What about the rest of the evidence array?

File: frontend/src/lib/interfaces/dispute.ts (hunk 1 above)
Comment (MichaelHorwitz) [reply frontend/src/lib/interfaces/dispute.ts:26]: Yes I think we need to change the actual contract

File: frontend/src/app/disputes/[id]/page.tsx (hunk 2 above)
Comment (MichaelHorwitz) [reply frontend/src/app/disputes/[id]/page.tsx:59]: Well when we can upload more than 1 evidence dock I'll change the functionality, shouldn't be too much

**Pull request merged**